Fix build on powerpc64 where we don't need this header and regular powerpc
doesn't use it.

Index: libretro-common/audio/audio_mix.c
--- libretro-common/audio/audio_mix.c.orig
+++ libretro-common/audio/audio_mix.c
@@ -24,8 +24,6 @@
 
 #if defined(__SSE2__)
 #include <emmintrin.h>
-#elif defined(__ALTIVEC__)
-#include <altivec.h>
 #endif
 
 #include <stdio.h>
